pandas/_libs/tslibs/ccalendar.pyi

```diff
@@ -8,5 +8,7 @@ def get_firstbday(year: int, month: int) -> int: ...
 def get_lastbday(year: int, month: int) -> int: ...
 def get_day_of_year(year: int, month: int, day: int) -> int: ...
 def get_iso_calendar(year: int, month: int, day: int) -> tuple[int, int, int]: ...
+def is_leapyear(year: int) -> bool: ...
 def get_week_of_year(year: int, month: int, day: int) -> int: ...
 def get_days_in_month(year: int, month: int) -> int: ...
+def dayofweek(y: int, m: int, d: int) -> int: ...
```

Comment:
`dayofweek` and `is_leapyear` are used only in pyx files

Reply:
no need to add. Python code cannot call these functions? With these added, Pylance will add to auto-completion. I don't think that's desirable.

Reply:
removed